projects/fitness_recommendations/requirements.txt

```diff
@@ -1,4 +1,149 @@
-flytekit>=0.30.3
-pandas
-s3fs
-sklearn
+aiobotocore==2.1.2
+aiohttp==3.8.1
+aioitertools==0.10.0
+aiosignal==1.2.0
+altair==4.2.0
+appnope==0.1.2
+argon2-cffi==21.3.0
+argon2-cffi-bindings==21.2.0
+arrow==1.2.2
+astor==0.8.1
+asttokens==2.0.5
+async-timeout==4.0.2
+attrs==21.4.0
+backcall==0.2.0
+base58==2.1.1
+binaryornot==0.4.4
+bleach==4.1.0
+blinker==1.4
+botocore==1.23.24
+cachetools==5.0.0
+certifi==2021.10.8
+cffi==1.15.0
+chardet==4.0.0
+charset-normalizer==2.0.12
+checksumdir==1.2.0
+click==8.0.4
+cloudpickle==2.0.0
+cookiecutter==1.7.3
+croniter==1.3.4
+dataclasses-json==0.5.6
+debugpy==1.5.1
+decorator==5.1.1
+defusedxml==0.7.1
+Deprecated==1.2.13
+diskcache==5.4.0
+docker==5.0.3
+docker-image-py==0.1.12
+docstring-parser==0.13
+entrypoints==0.4
+executing==0.8.3
+flyteidl==0.23.1
+flytekit==0.31.0
+frozenlist==1.3.0
+fsspec==2022.2.0
+gitdb==4.0.9
+GitPython==3.1.27
+googleapis-common-protos==1.55.0
+grpcio==1.44.0
+idna==3.3
+importlib-metadata==4.11.2
+ipykernel==6.9.1
+ipython==8.1.1
+ipython-genutils==0.2.0
+ipywidgets==7.6.5
+jedi==0.18.1
+Jinja2==3.0.3
+jinja2-time==0.2.0
+jmespath==0.10.0
+joblib==1.1.0
+jsonschema==4.4.0
+jupyter-client==7.1.2
+jupyter-core==4.9.2
+jupyterlab-pygments==0.1.2
+jupyterlab-widgets==1.0.2
+kaggle==1.5.12
+keyring==23.5.0
+MarkupSafe==2.1.0
+marshmallow==3.14.1
+marshmallow-enum==1.5.1
+marshmallow-jsonschema==0.13.0
+matplotlib-inline==0.1.3
+mistune==0.8.4
+multidict==6.0.2
+mypy-extensions==0.4.3
+natsort==8.1.0
+nbclient==0.5.12
+nbconvert==6.4.2
+nbformat==5.2.0
+nest-asyncio==1.5.4
+notebook==6.4.8
+numpy==1.22.3
+packaging==21.3
+pandas==1.4.1
+pandocfilters==1.5.0
+parso==0.8.3
+pexpect==4.8.0
+pickleshare==0.7.5
+Pillow==9.0.1
+poyo==0.5.0
+prometheus-client==0.13.1
+prompt-toolkit==3.0.28
+protobuf==3.19.4
+protoc-gen-swagger==0.1.0
+ptyprocess==0.7.0
+pure-eval==0.2.2
+py==1.11.0
+pyarrow==6.0.1
+pycparser==2.21
+pydeck==0.7.1
+Pygments==2.11.2
+Pympler==1.0.1
+pyparsing==3.0.7
+pyrsistent==0.18.1
+python-dateutil==2.8.2
+python-json-logger==2.0.2
+python-slugify==6.1.1
+pytimeparse==1.1.8
+pytz==2021.3
+pytz-deprecation-shim==0.1.0.post0
+pyzmq==22.3.0
+regex==2022.3.2
+requests==2.27.1
+responses==0.19.0
+retry==0.9.2
+s3fs==2022.2.0
+scikit-learn==1.0.2
+scipy==1.8.0
+semver==2.13.0
+Send2Trash==1.8.0
+six==1.16.0
+sklearn==0.0
+smmap==5.0.0
+sortedcontainers==2.4.0
+stack-data==0.2.0
+statsd==3.3.0
+streamlit==1.7.0
+terminado==0.13.3
+testpath==0.6.0
+text-unidecode==1.3
+threadpoolctl==3.1.0
+toml==0.10.2
+toolz==0.11.2
+tornado==6.1
+tqdm==4.63.0
+traitlets==5.1.1
+typer==0.4.0
+typing-inspect==0.7.1
+typing_extensions==4.1.1
+tzdata==2021.5
+tzlocal==4.1
+urllib3==1.26.8
+validators==0.18.2
+wcwidth==0.2.5
+webencodings==0.5.1
+websocket-client==1.3.1
+widgetsnbextension==3.5.2
+wrapt==1.14.0
+yarl==1.7.2
+zipp==3.7.0
```

Comment:
Are these dependencies for other libraries that we need or is it required to have all of these for the project? My bigger question is how did these libraries get added to the requirements file?